File: app/src/main/java/com/penseapp/acaocontabilidade/login/view/activities/LoginView.java
package com.penseapp.acaocontabilidade.login.view.activities;

/**
 * Created by Filip on 23/02/2016.
 */
public interface LoginView {
    void onSignUpFailure();
    void onSignUpSuccess();

    void onLoginSuccess();
    void onLoginFailure();

    void onUserLoggedIn();
    void onUserLoggedOut();
}

File: app/src/main/java/com/penseapp/acaocontabilidade/login/view/activities/LoginView.kt
package com.penseapp.acaocontabilidade.login.view.activities

/**
 * Created by Filip on 23/02/2016.
 */
interface LoginView {

    fun onSignUpFailure()

    fun onSignUpSuccess()


    fun onLoginSuccess()

    fun onLoginFailure()


    fun onPasswordResetSuccess()

    fun onPasswordResetFailure()


    fun onUserLoggedIn()

    fun onUserLoggedOut()
}
